src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -131,7 +131,9 @@ public static int MainCore(string workingDirectory, string[] args)
                 command.CurrentDirectory = workingDirectory;
                 if (command is DownloadCommandBase downloadCommandBase && downloadCommandBase.NoCache)
                 {
-                    console.WriteWarning(NuGetCommand.Log_RestoreNoCacheInformation);
+                    //Inform users that NoCache option is just for disabling HttpCache and
+                    //suggest them to use NoHttpCache instead, which does the same thing.
```

Comment:
Comments should explain what is not immediately understandable by reading the code. The next line is a Console.LogInformation so it's clear we're outputting a line to the console. We can also see what message is being output but following the reference to NuGetCommand.Log_RestoreNoCacheInformation.

Consider a more concise comment that explains briefly what context is missing:

```suggestion
                    // NoCache option is deprecated. Users should use NoHttpCache instead.
```